services/src/unit-test/java/uk/gov/moj/sdt/services/BulkFeedbackServiceTest.java

```diff
@@ -149,6 +112,15 @@ public void setUp() {
 
     @Test
     void testGetBulkFeedback() {
+
+        final IGlobalParameter globalParameterData = new GlobalParameter();
+        globalParameterData.setName(IGlobalParameter.ParameterKey.DATA_RETENTION_PERIOD.name());
+        globalParameterData.setValue("90");
+        when(mockGlobalParameterCache.getValue(IGlobalParameter.class,
+                                               IGlobalParameter.ParameterKey.DATA_RETENTION_PERIOD.name())).thenReturn(globalParameterData);
+        bulkFeedbackService = new BulkFeedbackService(mockBulkSubmissionDao, mockGlobalParameterCache);
+        bulkFeedbackService.setBulkSubmissionDao(mockBulkSubmissionDao);
```

Comment:
The call to setBulkSubmissionDao() is redundant and can be removed.  The bulk submission DAO is set by the constructor on the previous line.